### ISSUE TITLE
braker predictions has very low busco scores, although I already used 31 RNA samples

### ISSUE THREAD

**Issue by smallfishcui** [Gaius-Augustus/BRAKER#553]
Hi,

I fed 31 RNAseq samples to braker1, but the BUSCO score of braker1 is C:52.5%[S:26.9%,D:25.6%],F:11.6%,M:35.9%,n:4896.
How could it be possible? 
The braker2 result based on protein is: C:71.7%[S:36.3%,D:35.4%],F:6.3%,M:22.0%,n:4896
TSEBRA prediction is a little higher but still low  C:73.8%[S:38.8%,D:35.0%],F:6.1%,M:20.1%,n:4896. And Tsebra is even predicting less gene than  braker 1.
When I combine RNAseq and protein in the same run, the result is C:84.5%[S:39.8%,D:44.7%],F:5.8%,M:9.7%,n:4896
This is too low for further analysis. Can anyone help? my genome is 99% complete

thanks,
Cui

**Comment by KatharinaHoff** [comment]
What was your protein input data set?

**Comment by smallfishcui** [comment]
Hi Katharina,

I used the plant protein as protein references:
wget https://v100.orthodb.org/download/odb10_plants_fasta.tar.gz
tar xvf odb10_plants_fasta.tar.gz
cat plants/Rawdata/* > proteins.fasta
This is used in braker2 analysis. Is this right? 

thanks,
Cui

**Comment by KatharinaHoff** [comment]
Yes, you used a suitable protein database. 

Does your plant have a close relative in NCBI genomes from which you could take a reference protein set?

**Comment by smallfishcui** [comment]
Hi Katharina,

Yes, I used rice and sorghum bicolor as close relative in gemoma and get 93.9% of genes annotated. 
For braker, I eventually improved the analysis, by masking the regions with very high coverage (>1e6, which is located at the distal ends of a few chromosomes), and by using a genome with repeat-masked genome, and then did braker analysis using etp mode. I finally get 95% of the genes annotated. Apparently repeats are a big concern ....So I guess I can close this issue now.

Thanks,
Cui
